config/overlays/openshift/size-calculator/storage_size_calculator.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: storage-size-calculator
+  namespace: openshift-logging
+spec:
+  selector:
+    matchLabels:
+      name: loki-operator-size-calculator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: loki-operator-size-calculator
+    spec:
+      containers:
+        - command:
+          - /size-calculator
+          image: quay.io/sasagarw/loki-operator:v0.1.32
```

Comment:
If don't have separate quay repo for this one, let's create on under quay.io/openshift-logging